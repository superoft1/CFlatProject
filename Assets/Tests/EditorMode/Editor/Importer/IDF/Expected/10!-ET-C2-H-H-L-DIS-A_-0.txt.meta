fileFormatVersion: 2
guid: df435e454394395479c7c2e79592b0be
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
