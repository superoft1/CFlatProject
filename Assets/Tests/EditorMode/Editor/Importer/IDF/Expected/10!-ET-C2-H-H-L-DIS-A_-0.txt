WeldNeckFlange, , (0.0, 0.0, -1.0), (-137.9, 51.0, 1.3), False
(-137.9, 51.0, 1.2)
(-137.9, 51.0, 1.3)
FreeVertex: 1
Pipe, , (0.0, 0.0, 1.0), (-137.9, 51.0, 1.4), False
(-137.9, 51.0, 1.3)
(-137.9, 51.0, 1.5)
FreeVertex: 0
Elbow90, , (1.0, 0.0, 0.0), (-137.9, 51.0, 1.8), False
(-137.9, 51.0, 1.5)
(-138.2, 51.0, 1.8)
FreeVertex: 0
EccentricPipingReducer, , (-1.0, 0.0, 0.0), (-138.3, 51.0, 1.8), False
(-138.4, 51.0, 1.8)
(-138.2, 51.0, 1.8)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-138.5, 51.0, 1.8), False
(-138.4, 51.0, 1.8)
(-138.6, 51.0, 1.8)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, 1.0), (-138.6, 51.0, 1.8), False
(-138.6, 51.0, 1.8)
(-138.6, 51.0, 1.8)
(-138.6, 51.0, 2.0)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-138.6, 51.0, 2.0), False
(-138.6, 51.0, 2.0)
(-138.6, 51.0, 2.1)
FreeVertex: 0
GlobeValve, , (0.0, 0.0, 1.0), (-138.6, 51.0, 2.1), False
(-138.6, 51.0, 2.1)
(-138.6, 51.0, 2.1)
FreeVertex: 0
PressureGauge, PG_102_11_10_1070_1E, (0.0, 0.0, 1.0), (-138.6, 51.0, 2.2), False
(-138.6, 51.0, 2.2)
(-138.6, 51.0, 2.1)
FreeVertex: 1
Pipe, , (-1.0, 0.0, 0.0), (-138.6, 51.0, 1.8), False
(-138.6, 51.0, 1.8)
(-138.7, 51.0, 1.8)
FreeVertex: 0
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-138.8, 51.0, 1.8), False
(-138.8, 51.0, 1.8)
(-138.7, 51.0, 1.8)
FreeVertex: 0
WeldNeckFlange, , (1.0, 0.0, 0.0), (-138.9, 51.0, 1.8), False
(-138.8, 51.0, 1.8)
(-138.9, 51.0, 1.8)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-139.0, 51.0, 1.8), False
(-138.9, 51.0, 1.8)
(-139.1, 51.0, 1.8)
FreeVertex: 0
Elbow90, , (0.0, 0.0, -1.0), (-139.5, 51.0, 1.8), False
(-139.1, 51.0, 1.8)
(-139.5, 51.0, 2.2)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-139.5, 51.0, 3.2), False
(-139.5, 51.0, 2.2)
(-139.5, 51.0, 4.1)
FreeVertex: 0
Elbow90, , (0.0, -1.0, 0.0), (-139.5, 51.0, 4.5), False
(-139.5, 51.0, 4.1)
(-139.5, 51.4, 4.5)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-139.5, 54.1, 4.5), False
(-139.5, 51.4, 4.5)
(-139.5, 56.8, 4.5)
FreeVertex: 0
WeldNeckFlange, , (0.0, 1.0, 0.0), (-139.5, 56.9, 4.5), False
(-139.5, 56.9, 4.5)
(-139.5, 56.8, 4.5)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-139.5, 57.0, 4.5), False
(-139.5, 56.9, 4.5)
(-139.5, 57.0, 4.5)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-139.5, 57.0, 4.5), False
(-139.5, 57.0, 4.5)
(-139.5, 57.1, 4.5)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-139.5, 58.2, 4.5), False
(-139.5, 57.1, 4.5)
(-139.5, 59.3, 4.5)
FreeVertex: 0
Elbow90, , (0.0, 0.0, 1.0), (-139.5, 59.7, 4.5), False
(-139.5, 59.3, 4.5)
(-139.5, 59.7, 4.1)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-139.5, 59.7, 2.5), False
(-139.5, 59.7, 4.1)
(-139.5, 59.7, 0.9)
FreeVertex: 0
Tee, , (-1.0, 0.0, 0.0), (-139.5, 59.7, 0.6), True
(-139.7, 59.7, 0.6)
(-139.2, 59.7, 0.6)
(-139.5, 59.7, 0.9)
FreeVertex: 1
Pipe, , (-1.0, 0.0, 0.0), (-139.8, 59.7, 0.6), False
(-139.7, 59.7, 0.6)
(-139.9, 59.7, 0.6)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-139.9, 59.7, 0.6), False
(-139.9, 59.7, 0.6)
(-139.9, 59.7, 0.6)
(-139.9, 59.7, 0.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-139.9, 59.7, 0.4), False
(-139.9, 59.7, 0.5)
(-139.9, 59.7, 0.4)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-139.9, 59.7, 0.4), False
(-139.9, 59.7, 0.3)
(-139.9, 59.7, 0.4)
FreeVertex: 1
Pipe, , (-1.0, 0.0, 0.0), (-139.9, 59.7, 0.6), False
(-139.9, 59.7, 0.6)
(-140.0, 59.7, 0.6)
FreeVertex: 0
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-140.1, 59.7, 0.6), False
(-140.1, 59.7, 0.6)
(-140.0, 59.7, 0.6)
FreeVertex: 0
CheckValve, , (-1.0, 0.0, 0.0), (-140.2, 59.7, 0.6), True
(-140.3, 59.7, 0.6)
(-140.1, 59.7, 0.6)
FreeVertex: 0
WeldNeckFlange, , (1.0, 0.0, 0.0), (-140.3, 59.7, 0.6), False
(-140.3, 59.7, 0.6)
(-140.4, 59.7, 0.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-140.5, 59.7, 0.6), False
(-140.4, 59.7, 0.6)
(-140.5, 59.7, 0.6)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-140.5, 59.7, 0.6), False
(-140.5, 59.7, 0.6)
(-140.5, 59.7, 0.6)
(-140.5, 59.7, 0.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-140.5, 59.7, 0.4), False
(-140.5, 59.7, 0.5)
(-140.5, 59.7, 0.4)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-140.5, 59.7, 0.4), False
(-140.5, 59.7, 0.3)
(-140.5, 59.7, 0.4)
FreeVertex: 1
Pipe, , (-1.0, 0.0, 0.0), (-140.6, 59.7, 0.6), False
(-140.5, 59.7, 0.6)
(-140.7, 59.7, 0.6)
FreeVertex: 0
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-140.8, 59.7, 0.6), False
(-140.8, 59.7, 0.6)
(-140.7, 59.7, 0.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-140.8, 59.7, 0.6), False
(-140.8, 59.7, 0.6)
(-140.8, 59.7, 0.6)
FreeVertex: 0
WeldNeckFlange, , (1.0, 0.0, 0.0), (-140.9, 59.7, 0.6), False
(-140.8, 59.7, 0.6)
(-140.9, 59.7, 0.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-141.0, 59.7, 0.6), False
(-140.9, 59.7, 0.6)
(-141.1, 59.7, 0.6)
FreeVertex: 0
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-141.1, 59.7, 0.6), False
(-141.2, 59.7, 0.6)
(-141.1, 59.7, 0.6)
FreeVertex: 0
GateValve, , (-1.0, 0.0, 0.0), (-141.4, 59.7, 0.6), False
(-141.5, 59.7, 0.6)
(-141.2, 59.7, 0.6)
FreeVertex: 0
WeldNeckFlange, , (1.0, 0.0, 0.0), (-141.6, 59.7, 0.6), False
(-141.5, 59.7, 0.6)
(-141.6, 59.7, 0.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-141.7, 59.7, 0.6), False
(-141.6, 59.7, 0.6)
(-141.8, 59.7, 0.6)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-141.8, 59.7, 0.6), False
(-141.8, 59.7, 0.6)
(-141.8, 59.7, 0.6)
(-141.8, 59.7, 0.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-141.8, 59.7, 0.4), False
(-141.8, 59.7, 0.5)
(-141.8, 59.7, 0.4)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-141.8, 59.7, 0.4), False
(-141.8, 59.7, 0.3)
(-141.8, 59.7, 0.4)
FreeVertex: 1
Pipe, , (-1.0, 0.0, 0.0), (-141.9, 59.7, 0.6), False
(-141.8, 59.7, 0.6)
(-142.0, 59.7, 0.6)
FreeVertex: 0
WeldNeckFlange, , (-1.0, 0.0, 0.0), (-142.0, 59.7, 0.6), False
(-142.1, 59.7, 0.6)
(-142.0, 59.7, 0.6)
FreeVertex: 0
GateValve, , (-1.0, 0.0, 0.0), (-142.2, 59.7, 0.6), False
(-142.4, 59.7, 0.6)
(-142.1, 59.7, 0.6)
FreeVertex: 0
WeldNeckFlange, , (1.0, 0.0, 0.0), (-142.5, 59.7, 0.6), False
(-142.4, 59.7, 0.6)
(-142.5, 59.7, 0.6)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-142.6, 59.7, 0.6), False
(-142.5, 59.7, 0.6)
(-142.7, 59.7, 0.6)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-142.7, 59.7, 0.6), False
(-142.7, 59.7, 0.6)
(-142.7, 59.7, 0.6)
(-142.7, 59.7, 0.5)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-142.7, 59.7, 0.4), False
(-142.7, 59.7, 0.5)
(-142.7, 59.7, 0.4)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-142.7, 59.7, 0.4), False
(-142.7, 59.7, 0.3)
(-142.7, 59.7, 0.4)
FreeVertex: 1
Pipe, , (-1.0, 0.0, 0.0), (-142.8, 59.7, 0.6), False
(-142.7, 59.7, 0.6)
(-142.8, 59.7, 0.6)
FreeVertex: 0
Elbow90, , (0.0, 0.0, -1.0), (-143.2, 59.7, 0.6), False
(-142.8, 59.7, 0.6)
(-143.2, 59.7, 1.0)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-143.2, 59.7, 2.6), False
(-143.2, 59.7, 1.0)
(-143.2, 59.7, 4.1)
FreeVertex: 0
Elbow90, , (0.0, -1.0, 0.0), (-143.2, 59.7, 4.5), False
(-143.2, 59.7, 4.1)
(-143.2, 60.0, 4.5)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-143.2, 60.1, 4.5), False
(-143.2, 60.0, 4.5)
(-143.2, 60.2, 4.5)
FreeVertex: 0
Elbow90, , (1.0, 0.0, 0.0), (-143.2, 60.6, 4.5), False
(-143.2, 60.2, 4.5)
(-143.6, 60.6, 4.5)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-145.3, 60.6, 4.5), False
(-143.6, 60.6, 4.5)
(-147.0, 60.6, 4.5)
FreeVertex: 0
Tee, , (-1.0, 0.0, 0.0), (-147.2, 60.6, 4.5), True
(-147.4, 60.6, 4.5)
(-147.0, 60.6, 4.5)
(-147.2, 60.6, 4.7)
FreeVertex: 2
FreeVertexTotal: 9
